ironfish/src/telemetry/telemetry.ts

```diff
@@ -189,6 +189,11 @@ export class Telemetry {
         type: 'integer',
         value: this.chain.head.sequence,
       },
+      {
+        name: 'create_new_block_template_duration',
+        type: 'float',
+        value: this.metrics.mining_newBlockTemplate.rate5m,
+      },
```

Comment:
we already conditionally push this down below right? So we don't need this here.

Reply:
Removed